src/Testcontainers/Builders/Base64Provider.cs

```diff
@@ -88,5 +96,18 @@ public IDockerRegistryAuthenticationConfiguration GetAuthConfig(string hostname)
       _logger.DockerRegistryCredentialFound(hostname);
       return new DockerRegistryAuthenticationConfiguration(authProperty.Name, credential[0], credential[1]);
     }
+
+    [CanBeNull]
+    private static byte[] DecodeBase64String(string base64)
+    {
+      try
+      {
+        return Convert.FromBase64String(base64);
+      }
+      catch (FormatException)
+      {
+        return null;
+      }
```

Comment:
Should we write a log message in case of an exception? It is likely a misconfiguration?

Reply:
Good point! I added logs for every code path where the `auth` part can't be decoded in f79827f2ba376b4e29be8327d08f89961252442f.